GDJS/Runtime/CustomRuntimeObject.ts

```diff
@@ -32,7 +32,6 @@ namespace gdjs {
     _flippedX: boolean = false;
```

Comment:
Is `export type ObjectConfiguration`, above in the file, still valid/useful? (it has a `content: any`)

Reply:
`content`is only used by the generated code but I think it's important to understand where it comes from and it can be useful for tests.

Reply:
Ok! Because you removed content in some generated code I was not sure if it's used or not.

Reply:
Ok I see, this is still used in objectData, but not in the code generated for properties.